apis/shopper-login/shopper-login.raml

```diff
@@ -1002,7 +1002,7 @@ types:
               The `code_challenge` is created by SHA256 hashing the `code_verifier` and Base64 encoding the resulting hash.
 
               The `code_verifier` should be a high entropy cryptographically random string with a minimum of 43 characters and a maximum of 128 characters.
-            required: true
+            required: false
```

Comment:
Verified with Blair on the SLAS team that `code_challenge` is not a **required** parameter.

For private clients, we **OMIT** this parameter on purpose.

Reply:
@joeluong-sfcc I'll check in with the SLAS team on this

Reply:
Verified that this has already been changed in the upstream RAML👍